developer/src/kmc/src/activities/buildLdmlKeyboard.ts

```diff
@@ -0,0 +1,81 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as kmc from '@keymanapp/kmc-keyboard';
+import { KvkFileWriter, CompilerCallbacks } from '@keymanapp/common-types';
+import { NodeCompilerCallbacks } from '../util/NodeCompilerCallbacks.js';
+
+export function buildLdmlKeyboard(infile: string, options: any) {
+  // TODO-LDML: consider hardware vs touch -- touch-only layout will not have a .kvk
```

Comment:
true at what layer do we want to expose that? I guess the kvk will just be null?

Reply:
Not sure at this point; I think it will come out in the wash as we work on that.